.github/workflows/publish-to-live-pypi.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish Python 🐍 distributions 📦 to pypi
+
+on:
+  release:
+    types:
+      - published
```

Comment:
@timthelion In this workflow, the trigger is the publishing of a release.

Currently the project doesn't use this functionality, but I can assure you that it's much easier this way.

Releases are here; https://github.com/auto-mat/django-import-export-celery/releases

In there you can draft new releases (after a PR is merged) where you decide the tag name for the new version and can fill in the other details for the release. Though I find it easier to get github to auto-generate the changes which it does between the new tag and the one previous.

Here's an example of a release I've done this way; https://github.com/django-cms/django-sekizai/releases/tag/3.0.0

Once you hit "publish" at the bottom of the release, this workflow triggers and the package is built and sent to pypi.org for us all to install.